contents/blog/a-chat-with-sid.md

```diff
@@ -0,0 +1,93 @@
+---
+date: 2020-08-03
+title: The early days of GitLab - a chat with Sid Sijbrandij
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+---
+
+It gets pretty easy to idolize the superstars of tech. One of the coolest things we've learned is that many of the most successful founders will find time to talk with you. To put this in context, GitLab are one of the world's leading open core companies, worth \$2.75B. Kind of a big deal. We wanted to learn even more than we could discern from their handbook.
```

Comment:
Since this is in first person, it'd be good to specify who's talking, maybe with a little "By: James, CEO of PostHog" line at the top somewhere?